ctms/ingest.py

```diff
@@ -0,0 +1,86 @@
+from dataclasses import dataclass, fields
+from typing import Generator, List, Optional
+
+from pydantic import BaseModel
+from sqlalchemy.dialects.postgresql import insert
+from sqlalchemy.orm import Session
+
+from ctms.database import Base
+from ctms.models import AmoAccount, Email, FirefoxAccount, Newsletter, VpnWaitlist
+from ctms.schemas import (
+    AddOnsTableSchema,
+    EmailTableSchema,
+    FirefoxAccountsTableSchema,
+    NewsletterTableSchema,
+    VpnWaitlistTableSchema,
+)
+
+
+@dataclass
```

Comment:
Did you want to use the pydantic version here? https://pydantic-docs.helpmanual.io/usage/dataclasses/
Fine as is as well!

Reply:
That's true, I figured the minimal thing was good enough for this purpose. Maybe I'll reach for pydantic next time just for consistency